test/stdlib/SmallString.swift

```diff
@@ -1,4 +1,6 @@
-// RUN: %target-build-swift -Xfrontend -disable-access-control -module-name a %s -o %t.out -O
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s > %t/main.swift
+// RUN: %target-build-swift -Xfrontend -disable-access-control -module-name a %t/main.swift %S/../Inputs/SmallStringTestUtilities.swift -o %t.out -O
```

Comment:
Do you still need `-disable-access-control`? Because `-disable-access-control` is basically not guaranteed to work, especially with `-O`.

Reply:
Well, that's not new, just moved, but yes. This uses _SmallString

Reply:
I guess as long as it's just `@usableFromInline` things you use it should be okay…

Reply:
Maybe we should have a different frontend flag for that.